maintainers/maintainer-list.nix

```diff
@@ -1534,6 +1534,12 @@
     githubId = 185443;
     name = "Alexey Lebedeff";
   };
+  binsky = {
+    email = "timo@binsky.org";
+    github = "binsky08";
+    githubId = 30630233;
+    name = "Timo Triebensky";
+  };
```

Comment:
Move this into a separate commit `maintainers: add binsky`, since you're touch a different file/attribute.

Reply:
moved to commit [eca83bc](https://github.com/NixOS/nixpkgs/pull/167951/commits/eca83bc35d26795204a6b10593fbedadc7d256bc)